.github/workflows/ci.yaml

```diff
@@ -49,13 +49,13 @@ jobs:
           tar -xvf pack.tgz
 
       - name: Build
-        run: PACK_CMD=./pack make
+        run: PACK_CMD="$(pwd)/pack" make
```

Comment:
Or maybe I could just install `pack` to `/usr/local/bin` so it would be on PATH.